File: csharp/api/Dtos/VersionDtos.cs
using System.Text.Json.Serialization;

namespace csharp.Dtos;

public class GetCurrentVersionRequestDto
{
    [JsonPropertyName("key")] public string Key { get; init; } = string.Empty;
}

public class GetCurrentVersionResponseDto
{
    [JsonPropertyName("version")] public string Version { get; init; } = string.Empty;
}

File: kotlin/app/src/main/java/com/example/cityexplorer/data/dtos/VersionDtos.kt
package com.example.cityexplorer.data.dtos

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class GetCurrentVersionRequestDto(
    @SerialName("key")
    val key: String = ""
)

@Serializable
data class GetCurrentVersionResponseDto(
    @SerialName("version")
    val version: String = ""
)